flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/sink/KafkaCommitterTest.java

```diff
@@ -60,6 +61,8 @@ public void testRetryCommittableOnRetriableError() throws IOException {
                             new KafkaCommittable(PRODUCER_ID, EPOCH, TRANSACTIONAL_ID, recyclable));
             producer.resumeTransaction(PRODUCER_ID, EPOCH);
             List<KafkaCommittable> recovered = committer.commit(committables);
+            // FLINK-25531: force the producer to close immediately, else it would take 1 hour
+            producer.close(Duration.ZERO);
```

Comment:
Nit: I think it would be better to move this statement after the assertions to make clear it has nothing to do with the test behavior and only should increase the shutdown speed.